### ISSUE TITLE
Collabora doesn't display Documents (.odt, ...)

### ISSUE THREAD

**Issue by StefInP** [nextcloud/ios#2944]
### Steps to reproduce

1. Try to open a Document (.odt, .doc, or .docx)

### Expected behaviour

The document should be displayed.

### Actual behaviour

The Collabora interface launches but doesn't dysplay the document.
File formats KO : .odt, .docx, .doc
File formats OK : .ods, .ppt, .odp
All files open normally in the Web interface.
It seems to have appeared after upgrade server to 28.0.6.
Restart serveur : no change.
Clear cache iOS app : no change.

### Screenshots

If applicable, add a screenshot showing the issue.

### Logs

```
logs from the iOS app : 
[communication.log](https://github.com/user-attachments/files/15686372/communication.log)

```

### Environment data

**iOS version:**  iOS 17.5.1

**Nextcloud iOS app version:** 5.3.1

**Server operating system:**

**Web server:** Apache, nginx

**Database:**

**PHP version:**

**Nextcloud version:** 28.0.6 (AIO 8.3.0)

**Comment by janborg** [comment]
Updating Nextcloud to 29.0.2 fixed it for me

**Comment by StefInP** [comment]
@janborg Thanks for the feed-back. I'll probably upgrade too.

**Comment by StefInP** [comment]
After updating Nextcloud to 29.0.2 (AIO 9.0.1), it still doesn't work for me, for .odt files.
No problem with .odp and .ods files that I can open and edit with Collabora in Nextcloud iOS app.
I have uninstalled and reinstalled iOS app. : no change.
[communication.log](https://github.com/user-attachments/files/15872904/communication.log)

**Comment by thebearon** [comment]
This should be fixed by CODE/Nextcloud Office 24.04.4-2.
